packages/mdc-chips/_mixins.scss

```diff
@@ -23,6 +23,7 @@
 @import "@material/ripple/mixins";
 @import "@material/theme/functions";
 @import "@material/theme/mixins";
+@import "./variables";
```

Comment:
Unused variables import.

Reply:
It's actually used in a few places in this file, was missing before.